build.gradle

```
@@ -1,31 +1,51 @@
 plugins {
-	id 'org.springframework.boot' version '2.2.5.RELEASE'
-	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-	id 'java'
+    id 'org.springframework.boot' version '2.2.5.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+    id "org.asciidoctor.convert" version "1.5.9.2"
+    id 'java'
 }
 
 group = 'com.example'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '1.8'
 
 repositories {
-	mavenCentral()
+    mavenCentral()
 }
 
 dependencies {
-	implementation 'org.springframework.boot:spring-boot-starter-web'
-	implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
-	implementation 'net.rakugakibox.spring.boot:logback-access-spring-boot-starter:2.7.1'
-	implementation 'pl.allegro.tech.boot:handlebars-spring-boot-starter:0.3.0'
-	implementation 'org.jgrapht:jgrapht-core:1.0.1'
-	implementation 'io.jsonwebtoken:jjwt:0.9.1'
-	testImplementation 'io.rest-assured:rest-assured:3.3.0'
-	testImplementation('org.springframework.boot:spring-boot-starter-test') {
-		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
-	}
-	runtimeOnly 'com.h2database:h2'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
+    implementation 'net.rakugakibox.spring.boot:logback-access-spring-boot-starter:2.7.1'
+    implementation 'pl.allegro.tech.boot:handlebars-spring-boot-starter:0.3.0'
+    implementation 'org.jgrapht:jgrapht-core:1.0.1'
+    implementation 'io.jsonwebtoken:jjwt:0.9.1'
+    asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:2.0.4.RELEASE'
+    testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc:2.0.4.RELEASE'
```

```suggestion
    asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor'
    testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
```

plugins에서 'io.spring.dependency-management'가 어떠한 역할을 하는지 공부해보세요~

---

`dependency-management 플러그인`은 현재 사용하고 있는 Spring Boot의 버전에 맞게 사용하고자 하는 라이브러리의 버전을 자동으로 가져오는 역할을 하고, 사용하려는 의존성을 관리해주는 플러그인이라고 공부했습니다. :)

아래의 글을 보고 학습했습니다.
https://docs.spring.io/dependency-management-plugin/docs/current/reference/html/
https://dingue.tistory.com/17